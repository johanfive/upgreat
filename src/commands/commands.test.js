const plan = require('./plan')
const up = require('./up')

it('exports plan', () => {
  expect(plan).toMatchSnapshot()
})

it('exports up', () => {
  expect(up).toMatchSnapshot()
})
