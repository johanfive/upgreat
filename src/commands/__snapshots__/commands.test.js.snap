// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports plan 1`] = `[Function]`;

exports[`exports up 1`] = `[Function]`;
